spec.md

```diff
@@ -591,13 +595,41 @@ schema being used.
 }
 ```
 
+##### Deprecation Object 
+
+| Response Field | Type | Description |
+| --- | --- | --- |
+| deprecated* | boolean | Indicates that this service or plan is deprecated and will be removed in the future. See [deprecating services and plans](#deprecating-services-and-plans) for more information. |
+| description | string | A short description of the deprecation. |
+| since | date string | ISO 8601 formatted date when the Service Broker deprecated the service or plan. |
+| eol | date string | ISO 8601 formatted date when the Service Broker expects to remove the service or plan from the catalog. |
+| alterntives | array of strings | An array of alterntives services or plans that should be concidered from the broker. This is a list of names. If the debrecated object is a service, these are service names. If the debrecated object is a plan, these are plan names within the same service. |
+
+\* Fields with an asterisk are REQUIRED.
 
 ### Adding a Service Broker to the Platform
 
 After implementing the first endpoint `GET /v2/catalog` documented
 [above](#catalog-management), the Service Broker will need to be registered
 with your Platform to make your services and plans available to end users.
 
+### Deprecating services and plans
+
+A Service Broker can signal to a Platform (through the Catalog) that a service
+or plan is deprecated. The Service Broker MAY provide optional messaging and
+alterntives for the deprecated service or plan. This gives the Platform the
+opportunity to display meaningful advice to their users to migrate away from
+the deprecated service or plan.
+
+Service Brokers MUST allow existing Service Instances to be updated as they
```

Comment:
not just "updated" but also existing Instances and Bindings of those services/plans MUST still be useable.